doc/changelog.md

```diff
@@ -157,78 +293,72 @@
   [{'expval0': (array(-0.0978434), array(-0.19767681))}, (array(-0.0978434), array(-0.19767681))]
   ```
 
-* Add native support for `qml.PSWAP` and `qml.ISWAP` gates on Amazon Braket devices. Specifically, a circuit like
-
-  ```py
-  import pennylane as qml
-  from catalyst import qjit
+* Support has been added for linear algebra functions which depend on computing the eigenvalues
+  of symmetric matrices, such as `np.sqrt_matrix()`.
```

Comment:
```suggestion
* Support has been added for linear algebra functions which depend on computing the eigenvalues
  of symmetric matrices at run time backed by `lapack_dsyevd`, such as `np.sqrt_matrix()`.
```

Reply:
@maliasadi I moved the mention of lapack/DSYEVD below :)